_data/sidebars/home_sidebar.yml

```diff
@@ -154,6 +154,30 @@ entries:
           url: /participants.html
           output: web
 
+      - title: FAQs for Collections
+        output: web
+        subfolderitems:
+
+        - title: "FAQ Collection Discovery"
+          url: /collection_extensions_faq_discovery.html
```

Comment:
Do these links work?  I'm not seeing where these files exist?

Reply:
These URLs did work.  They are defined in the frontmatter of the associated md file.

FYI... the URLs changed anyway because I did a bit of reorganizing while proof-reading and making your suggested changed.